parts/linux/cloud-init/artifacts/health-monitor.sh

```diff
@@ -6,14 +6,27 @@
 set -o nounset
 set -o pipefail
 
+# node could very well be running both dockerd and containerd. we just care about the runtime that kubelet is using.
+getKubeletRuntime() {
```

Comment:
can we not decide this when generating the script?

Reply:
good point. i'll template this out in a separate pr.

Reply:
TODO: separate PR for cse